recipes-test/demo-config/secondary-config.bb

```diff
@@ -12,14 +12,14 @@ SOTA_HARDWARE_ID ?= "${MACHINE}-sndry"
 SECONDARY_HARDWARE_ID ?= "${SOTA_HARDWARE_ID}"
 
 SRC_URI = "\
-    file://30-fake-pacman.toml \
+    file://30-ostree-pacman.toml \
```

Comment:
What will happen to `30-fake-pacman.toml`? Will it be unused?

Reply:
I don't know, it depends on our decision on the file/binary update configuration.

Reply:
We're only testing ostree in CI, and if there is no demo or test for the fake package manager on the secondary, I would prefer to remove the unused file.

Reply:
ok